akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Plugins.scala

```diff
@@ -216,44 +236,87 @@ object PluginModel {
+          } else {
+            val errorMessage = s"Cannot load plugin [$pluginName]:" +
+              s" system SDK version [$html5PluginSdkVersion] does not satisfy plugin" +
+              s" requirement [${pluginObjectWithAbsoluteUrls.manifest.content.get.requiredSdkVersion}]."
+            logger.error(errorMessage)
+            pluginsMap = pluginsMap + (
+              pluginName -> pluginObjectWithAbsoluteUrls.copy(
+                loadFailureReason = Some(errorMessage), loadFailureSource = Some("bbb-apps-akka")
+              ))
+          }
+        } else pluginsMap = pluginsMap + (pluginName -> Plugin(
+          PluginManifest(
+            pluginObject.manifest.url,
+            None,
+          ),
+          pluginObject.loadFailureReason,
+          pluginObject.loadFailureSource
+        ))
       } catch {
-        case err @ (_: JsonProcessingException | _: JsonMappingException) => logger.error(
-          s"Error while processing plugin $pluginName: $err"
-        )
-        case err @ (_: NumberFormatException) => logger.error(
-          s"Plugin SDK version of either HTML5 or plugin manifest couldn't be parsed ($pluginName): $err"
-        )
+        case err @ (_: JsonProcessingException | _: JsonMappingException) =>
+          val errorMessage = s"Error while processing plugin [$pluginName]: $err"
+          pluginsMap = pluginsMap + (pluginName -> createEmptyPluginWithAkkaError(errorMessage))
+          logger.error(errorMessage)
+        case err @ (_: NumberFormatException) =>
+          val errorMessage = s"Plugin SDK version of either HTML5 or plugin manifest couldn't be parsed [$pluginName]: $err"
+          pluginsMap = pluginsMap + (pluginName -> createEmptyPluginWithAkkaError(errorMessage))
+          logger.error(errorMessage)
+        case err @ (_: PluginHtml5VersionValidationException) =>
+          val errorMessage = err.message
+          pluginsMap = pluginsMap + (pluginName -> createEmptyPluginWithAkkaError(errorMessage))
+        case err @ (_: Exception) =>
+          val errorMessage = s"Generic exception in akka for plugin [$pluginName]: ${err.getMessage}"
+          pluginsMap = pluginsMap + (pluginName -> createEmptyPluginWithAkkaError(errorMessage))
+          logger.error(errorMessage, err)
       }
     }
     instance.plugins = pluginsMap
     validatePluginsBeforeCreateModel(instance, clientSettings)
   }
+
   def persistPluginsForClient(meetingId: String, instance: PluginModel): Unit = {
-    instance.plugins.foreach { case (_, plugin) =>
-      PluginDAO.insert(meetingId, plugin.manifest.content.name, plugin.manifest.content.javascriptEntrypointUrl,
-        plugin.manifest.content.javascriptEntrypointIntegrity.getOrElse(""), plugin.manifest.content.localesBaseUrl)
+    instance.plugins.foreach { case (pluginNameRaw, plugin) =>
+      val pluginName = if (plugin.manifest.url == pluginNameRaw) "NOT-USED" else pluginNameRaw
```

Comment:
```suggestion
      val pluginName = if (plugin.manifest.url == pluginNameRaw) "unidentified-plugin" else pluginNameRaw
```

The name `NOT-USED` is a bit cofusing:
<img width="1859" height="478" alt="image" src="https://github.com/user-attachments/assets/355d3ef4-4ad7-496d-8661-50a3a021d188" />